eventdemo/resources/chatEventClassFamily.json

```diff
@@ -0,0 +1,53 @@
+[
```

Comment:
Why u add chatEventClassFamily.json? We have chat_event_class_event.avsc and chat_event_class_message.avsc in eventdemo/resources folder